std/stdio.d

```diff
@@ -1522,6 +1524,143 @@ for every line.
     }
 
 /**
+Creates a file with a randomly generated name and returns an open File to it.
+
+The name will start with the given prefix and be located in the given directory.
+If no directory is given, then it will default to the result of
+$(XREF file, tempDir).
+
+There is nothing special about the file created other than the fact that its
+name is randomly generated, and it defaults to being in the temp directory on
+the system. It will only be deleted if explicitly deleted or if it's in the temp
+directory, and the temp is cleared out (as happens on system startup or shutdown
+on some systems).
+
+Params:
+    prefix = Optional prefix for file name.
+    suffix = Optional suffix for file name (which also provides a way to give
+             the file an extension).
+    dir    = Directory of temporary file.
```

Comment:
I don't follow. If you want to create a temp file in the current working directory, then just provide that directory to `tempFile` - e.g. `File.tempFile("", "", std.file.getcwd())`. What's the potential source of unexpected behavior?